clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/security/AppEngineCredentialsInitializer.groovy

```diff
@@ -68,8 +78,11 @@ class AppEngineCredentialsInitializer implements CredentialsInitializerSynchroni
           .applicationName(clouddriverUserAgentApplicationName)
           .jsonPath(managedAccount.jsonPath)
           .requiredGroupMembership(managedAccount.requiredGroupMembership)
+          .serviceAccountEmail(managedAccount.serviceAccountEmail)
           .build()
 
+        jobExecutor.runCommand(["gcloud", "auth", "activate-service-account", "--key-file", managedAccount.jsonPath])
```

Comment:
Previously, wasn't it possible to start this using just application default credentials?